metricbeat/module/apache/_meta/Dockerfile

```diff
@@ -1,2 +1,4 @@
 FROM httpd:2.4.20
+RUN apt-get update && apt-get install -y curl
```

Comment:
That will slow down the builds quite a bit if we have it several times. But I don't have a better solution.